app/assets/stylesheets/orders.css.scss

```diff
@@ -22,14 +22,252 @@
   }
 }
 
-.payment-form {
-  width: 30%;
+.order-summary {
+  @include border-bottom-radius($base-border-radius);
+  @include border-top-radius($base-border-radius);
+  @include span-columns(6);
 
-  .alert {
-    width: 950px;
+  border: $base-border;
+
+  &:after,
+  &:before {
+    content: "";
+    display: table;
+  }
+
+  .section {
+    @include fill-parent;
+
+    border-top: $base-border;
+    margin-top: 0;
+    padding: 1.5em;
+
+    &:first-child {
+      border-top: none;
+    }
+  }
+
+  h3 {
+    margin: 0;
+  }
+
+  img {
+    border-radius: $base-border-radius;
+    float: left;
+    margin: 0 0 0 -6em;
+    width: 5em;
+  }
+
+  li {
+    border-top: $base-border;
+    padding: 1.5em 5.5em 1.5em 6em;
+
+    &:after {
+      clear: both;
+    }
+
+    &:after,
+    &:before {
+      content: "";
+      display: table;
+    }
+
+    &:first-child {
+      border-top: none;
+      margin-top: -1.5em;
+    }
+
+    &:last-child {
+      margin-bottom: -1.5em;
+    }
+  }
+
+  ul {
+    margin: 0;
+  }
+
+  .info {
+    float: left;
+    margin-top: .4em;
+  }
+
+  .description, .name {
+    display: block;
+  }
+
+  .name {
+    margin-bottom: .4em;
+  }
+
+  .price {
+    margin-right: -5.5em;
+    float: right;
+    margin-top: .4em;
+  }
+
+  .total {
+    border-top: $base-border;
+    margin-top: 1em;
+    padding-top: 1em;
+  }
+}
+
+.payment-due-container {
+  &:after {
+    clear: both;
+  }
+
+  &:after,
+  &:before {
+    content: "";
+    display: table;
+  }
+}
+
+.payment-due__label {
+  float: left;
+  padding: .5em 0;
+  margin: 0;
+}
+
+.payment-due {
+  line-height: 1em;
+
+  &:after {
+    clear: both;
+  }
+
+  &:after,
+  &:before {
+    content: "";
+    display: table;
+  }
+}
+
+.payment-due__price {
+  float: right;
+  color: #8fc243;
```

Comment:
Color literals like `#8fc243` should only be used in variable declarations; they should be referred to via variable everywhere else.